README.md

```diff
@@ -25,3 +23,37 @@ ds = open_dataset(
     branch="my_new_testdata_branch"
 )
 ```
+
+> **Note**
+> The following options only work for branches based on `Ouranosinc/xclim-testdata`, not forks.
+
+If you wish to run the entire `xclim` testing suite locally against your branch, this can be set via an environment variable:
+```shell
+$ export XCLIM_TESTDATA_BRANCH="my_new_testdata_branch"
+
+$ pytest xclim
+# or, alternatively:
+$ tox
+```
+
+If you wish to run the entire `xclim` testing suite on the `Ouranosinc/xclim` GitHub Workflows (CI) against your branch,
+this can be set via an environment variable default in the `.github/workflows/main.yml` workflow configuration:
+```yaml
+env:
+  XCLIM_TESTDATA_BRANCH: my_new_testdata_branch
+```
+
+> **Note**
+> Be aware that modifying this variable to a value other than the latest tagged version of `xclim-testdata`
+> will trigger a GitHub Workflow that will block merging of your Pull Request until changes are effected.
+
+## Versioning
+When updating a dataset in `xclim-testdata` using a development branch and Pull Request, once changes have been merged
+to the `main` branch, you should tag a new version of `xclim-testdata`. The version tag of `xclim-testdata` is loosely tied
+to that of `xclim`, and should represent the `xclim` release (**stable**, i.e. version string not ending in `-dev`)
```

Comment:
"version tag of `xclim-testdata` is loosely tied to that of `xclim`"

Does that means each new "stable" version of `xclim` will have a corresponding same tag in `xclim-testdata`?  So we are forced to tag in `xclim-testdata` even if no data changed between 2 stable releases of `xclim`?

That sounds like extra work and easy to forget because no data changed in `xclim-testdata`.

Reply:
Revised: Tagging should follow from calendar versioning schemes. Changes need to be made in xclim to reflect this.